packages/native-stack/src/views/NativeStackView.native.tsx

```diff
@@ -136,7 +137,7 @@ type SceneViewProps = {
   isPresentationModal?: boolean;
   onWillDisappear: () => void;
   onWillAppear: () => void;
-  onAppear: () => void;
+  onAppear: ScreenProps['onAppear'];
   onDisappear: () => void;
```

Comment:
Just for clarity, can we propagate this change onto other events?

Reply:
@tboba yes, sure. I'll make these changes and push new changes 👍